### PR TITLE
fix: responsive long url in plaintext

### PR DESCRIPTION
**Pull request by MickGe**
This PR fixes 

## Changes

* Long URL in plaintext breaks responsive design on small screen

### CODE REVIEW

File: css/bootstrap/privatebin.css [hunk 1]
```diff
@@ -87,6 +87,10 @@ body.loading {
 	word-wrap: break-word;
 }
 
+#plaintext > a {
```
Comment (elrido) [inline css/bootstrap/privatebin.css:90]: In the interest of not repeating yourself, how about merging this with the rule above on line 82?
```suggestion
#pastelink > a, #plaintext > a {
```

File: css/bootstrap5/privatebin.css [hunk 2]
```diff
@@ -32,6 +32,10 @@
 	word-wrap: break-word;
 }
 
+#plaintext > a {
```
Comment (elrido) [inline css/bootstrap5/privatebin.css:35]: Same DRY suggestion here, merge with the rule above on 31:
```suggestion
#pastelink > a, #plaintext > a {
```

**Review by elrido — CHANGES_REQUESTED** [review]
Thank you for submitting this, definitely a worthwhile quality of life improvement. I would suggest to merge this with the block just before, as they are the same rule, just a different target.

File: css/bootstrap/privatebin.css (hunk 1 above)
Comment (MickGe) [reply css/bootstrap/privatebin.css:90]:
```suggestion
#pastelink > a, #plaintext  a,  {
```

I made a mistake…

File: css/bootstrap5/privatebin.css (hunk 2 above)
Comment (MickGe) [reply css/bootstrap5/privatebin.css:35]:
```suggestion
#pastelink > a, #plaintext  a,  {
```

I made a mistake…

File: css/bootstrap/privatebin.css (hunk 1 above)
Comment (elrido) [reply css/bootstrap/privatebin.css]: That wasn't quite what I had in mind, but I wanted to get this merged, so I resolved the conflict in the merge commit into master. Also realized this (and some other selectors) could get de-duplicated into the common.css.

**Pull request merged**